Jenkinsfile

```diff
@@ -14,6 +14,7 @@ pipeline {
         GIT = credentials('65325e52-5ec0-46a7-a937-f81f545f3c1b')
       }
       steps {
+        slackSend color: 'warning', channel: '#dev', message: "Android app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} is awaiting Release Notes input to finish Build"
```

Comment:
Please change the last part this way: "is awaiting release notes input to start the build"

Reply:
Done